index.js

```diff
@@ -24,14 +24,17 @@ function CacheBuster(options) {
         return path.join(dirname, basename + '.' + checksum + extname);
     };
 
-    this.mappings = {};
+    this.mappings = {
+        baseDir: null,
```

Comment:
In order to construct the relative paths, I need to keep a reference to the baseDir that was used to construct the mappings.